bciers/apps/reporting/src/app/components/finalReview/templates/ActivityView.tsx

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+import SourceTypeBoxTemplate from "@bciers/components/form/fields/SourceTypeBoxTemplate";
+
+const styles = {
+  sourceCard: {
+    backgroundColor: "white",
+    padding: "16px",
+    marginBottom: "20px",
+    borderRadius: "8px",
+    boxShadow: "0 1px 4px rgba(0,0,0,0.1)",
+  },
+  dataCard: {
+    backgroundColor: "#f2f2f2",
```

Comment:
some of the styling here could be cleaned up here, for example in `bciers/libs/styles/src/colors.ts` we have a bunch of hex colour codes set as variable names. (#f2f2f2 = `BC_GOV_BACKGROUND_COLOR_GREY`, 003366 = `BC_GOV_PRIMARY_BRAND_COLOR_BLUE`) that could make our future lives easier.